Userland/Libraries/LibGUI/Application.cpp

```diff
@@ -224,7 +224,7 @@ void Application::request_tooltip_show()
     int const margin = 30;
     Gfx::IntPoint adjusted_pos = ConnectionToWindowServer::the().get_global_cursor_position();
 
-    adjusted_pos.translate_by(0, 18);
+	adjusted_pos.translate_by(0, 14); //Adjusted from 0,18 7/25/2022
```

Comment:
The indentation is wrong here, and this `//Adjusted from 0,18 7/25/2022` comment is unnecessary.